getl/blocks/load/entrypoint.py

```diff
@@ -34,7 +34,7 @@ def batch_parquet(conf: BlockConfig) -> DataFrame:
     SectionName:
         Type: load::batch_parquet
         Properties:
-            Path: s3://husvarna-datalake/trusted/amc/live
+            Path: s3://husqvarna-datalake/trusted/amc/live
```

Comment:
Wops, we should probably rename this to s3://bucket/path/to/files because its an open source repo.